pkg/resource/logging-config/alloy/logging-config.alloy.yaml.template

```diff
@@ -58,3 +61,60 @@ alloy:
           path: /run/log/journal
       - name: alloy-tmp
         emptyDir: {}
+podLogs:
+{{- if .IsWorkloadCluster }}
+- name: namespace-default
+  namespace: giantswarm
+  spec:
+    selector: {}
+    namespaceSelector:
+      matchExpressions:
+      - key: kubernetes.io/metadata.name
+        operator: In
+        values:
+        - kube-system
```

Comment:
@giantswarm/team-shield i'm guessing WE should add kyverno namespace so.we Can collect the kyverno logs?

Reply:
Yes it would be super useful to have the Kyverno logs :slightly_smiling_face: 
What did Atlas have in mind about ownership/config for that label? I would say it might make sense to have this list of namespaces set through `shared-configs` or something similar instead of being baked into logging-operator, but we could also set it inside the Kyverno app if Atlas wanted to leave it up to the teams

Reply:
The idea is that we would set a static list for well static namespace liké kyverno and kube-system and the rest would bé for customers to set up using labels. Maybe this could bé externalités via config sure

Reply:
Yes I agree we should move this list out of logging-operator

Reply:
I added the following Helm chart value, this can be overwritten via usual mechanism like shared-configs
```
loggingOperator:
  defaultWorkloadClusterNamespaces: "kube-system,giantswarm"
```